### PR TITLE
Style Guide Consistency Updates

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation ticket: https://mattermost.atlassian.net/browse/MM-31518

Updated:
- Minor Style Guide updates

### CODE REVIEW

File: source/cloud/cloud-integrations/cloud-slash-commands.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ Suppose you want to write an external application that is able to check the weat
 You can follow these general guidelines to set up a custom Mattermost slash command for your application.
 
 1. Open **Main Menu > Integrations > Slash Commands**. If you don't have the **Integrations** option in your Main Menu, slash commands may not be enabled or may be disabled for non-admins. Enable them from **System Console > Integrations > Integration Management** or ask your Mattermost System Admin to do so.
-2. Select **Add Slash Command**, then add a name and description for the command.
+2. Select **Add Slash Command**, then add name and description for the command.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-slash-commands.rst:64]:
```suggestion
2. Select **Add Slash Command**, then add a name and description for the command.
```

File: source/cloud/cloud-integrations/cloud-slash-commands.rst [hunk 2]
```diff
@@ -147,14 +147,14 @@ Tips and Best Practices
...
+4. The external application may be written in any programming language. It needs to provide a URL which receives the request sent by your Mattermost workspace and responds with in the required JSON format.
 
 Share Your Integration
 ----------------------
 
-If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://integrations.mattermost.com/>`__.
+If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://about.mattermost.com/default-app-directory/>`__.
 
-The `app directory <https://integrations.mattermost.com/>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or self-managed infrastructure.
+The `app directory <https://about.mattermost.com/default-app-directory/>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or on-prem infrastructure.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-slash-commands.rst:157]:
```suggestion
The `app directory <https://about.mattermost.com/default-app-directory>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or self-managed infrastructure.
```

File: source/cloud/cloud-integrations/cloud-slash-commands.rst [hunk 3]
```diff
@@ -147,14 +147,14 @@ Tips and Best Practices
...
-4. The external application may be written in any programming language. It needs to provide a URL which receives the request sent by your Mattermost server and responds with in the required JSON format.
+4. The external application may be written in any programming language. It needs to provide a URL which receives the request sent by your Mattermost workspace and responds with in the required JSON format.
 
 Share Your Integration
 ----------------------
 
-If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://integrations.mattermost.com/>`__.
+If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://about.mattermost.com/default-app-directory/>`__.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-slash-commands.rst:155]:
```suggestion
If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects/>`__ in our `app directory <https://about.mattermost.com/default-app-directory>`__.
```

File: source/cloud/cloud-integrations/cloud-slash-commands.rst [hunk 4]
```diff
@@ -188,4 +188,4 @@ Known Slack compatibility issues
 Troubleshooting
 ---------------
 
-See `developer documentation <https://developers.mattermost.com/integrate/slash-commands/>`__ for troubleshooting, or `join the Mattermost user community for help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
+See `developer documentation <https://developers.mattermost.com/integrate/slash-commands>`__ for troubleshooting, or `join the Mattermost user community for help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-slash-commands.rst:191]: Should the second URL have a trailing slash?

**Review by justinegeffen — COMMENTED** [review]
Some minor corrections and a question. :)

File: source/cloud/cloud-integrations/cloud-slash-commands.rst (hunk 4 above)
Comment (cwarnermm) [reply source/cloud/cloud-integrations/cloud-slash-commands.rst:191]: It appears to resolve with and without the trailing slash.

**Review by justinegeffen — APPROVED** [review]

File: source/cloud/cloud-integrations/cloud-slash-commands.rst [hunk 5]
```diff
@@ -4,9 +4,9 @@ Slash Commands
 
 .. note::
 
-  This is the admin documentation for slash commands. If you're a developer looking to build an integration, see `our developer documentation <https://developers.mattermost.com/integrate/getting-started/>`__.
+  This is the admin documentation for slash commands. If you're a developer looking to build an integration, see `our developer documentation <https://developers.mattermost.com/integrate>`__.
```
Comment (amyblais) [inline source/cloud/cloud-integrations/cloud-slash-commands.rst:7]: This links is "broken" (the page is empty).

Comment (cwarnermm) [reply source/cloud/cloud-integrations/cloud-slash-commands.rst:7]: Thanks, @amyblais. Fixed via https://github.com/mattermost/docs/pull/4426

**Pull request merged**